src/egon/data/datasets/power_plants/wind_farms.py

```diff
@@ -313,7 +326,9 @@ def match_district_se(x):
     if fed_state in north:
         state_wf["inst capacity [MW]"] = power_north * state_wf["area [km²]"]
     else:
-        state_wf["inst capacity [MW]"] = power_south * state_wf["area [km²]"]
+        state_wf["inst capacity [MW]"] = (
+            power_south * state_wf["area [km²]"]
+        )  # ToDo: fix this when scn eGon100RE
```

Comment:
Is this still an open todo?

Reply:
Nice that you mentioned it. This ToDo is now fixed in a0dc3fd7987bd31b82b42c2463fe181bf2263d0d